lib/gen_rmq/consumer/queue_configuration.ex

```diff
@@ -102,35 +46,10 @@ defmodule GenRMQ.Consumer.QueueConfiguration do
     create_dead_letter = dead_letter[:create]
 
     options
-    |> setup_ttl()
-    |> setup_priority()
     |> setup_dead_letter_exchange(dead_letter, create_dead_letter)
     |> setup_dead_letter_routing_key(dead_letter, create_dead_letter)
   end
 
-  defp setup_ttl(options) do
-    case options[:ttl] do
-      nil ->
-        options
-
-      value ->
-        args = Keyword.get(options, :arguments, [])
-        Keyword.put(options, :arguments, [{"x-expires", :long, value} | args])
-    end
-  end
-
-  defp setup_priority(options) do
-    case options[:max_priority] do
-      nil ->
-        options
-
-      value ->
-        value = set_max_priority_to_highest_value(value)
-        args = Keyword.get(options, :arguments, [])
-        Keyword.put(options, :arguments, [{"x-max-priority", :long, value} | args])
-    end
-  end
```

Comment:
Something similar is needed since the x-max-priority argument value should be between 1 and 255, so we should call the 'set_max_priority_to_highest_value' to check if the value is above the limit.

Reply:
all right, will do smth similar 👍

Reply:
@Shemeikka should be fixed in the latest commit -> https://github.com/meltwater/gen_rmq/pull/221/commits/36b205e7f294e2dc899f32d8625b49af670ebbc9

Reply:
Looks good